docs/Text.md

```diff
@@ -0,0 +1,185 @@
+	* "L": left
+	* "T": top
+	* "R": right
+	* "B": bottom
+* __align__ (str): Allows to center or align the text inside the cell. Possible values are:
+	* "L" or empty string: left align
+	* "C": center 
+	* "R": right align
+	* "J": justify
+* __fill__ (bool): Indicates if the cell background must be painted (`True`)
+	or transparent (`False`).
+* __split_only__ (bool): if `True`, don't output anything, only perform
+	word-wrapping and return the resulting multi-lines array of strings.
+* __link__ (str): optional link to add on the cell, internal
+	(identifier returned by `add_link`) or external URL.
+* __new_x__ (Enum XPos): New current position in x after the call.
+* __new_y__ (Enum YPos): New current position in y after the call.
+* __max_line_height__ (float): optional maximum height of each sub-cell generated
+* __markdown__ (bool): enable minimal markdown-like markup to render part
+	of text as bold / italics / underlined.
+* __print_sh__ (bool): Treat a soft-hyphen (\\u00ad) as a normal printable
+	character, instead of a line breaking opportunity.
+
+
+## .write()
+Prints multi-line text between the page margins, starting from the current position.
+When the right margin is reached, a line break occurs at the most recent
+space or soft-hyphen character, and text continues from the left margin.
+A manual break happens any time the \\n character is met,
+Upon method exit, the current position is left near the end of the text, ready for the next call to continue without a gap, potentially with a different font or size set. Returns a boolean indicating if page break was triggered.
+
+The primary purpose of this method is to print continuously wrapping text, where different parts may be rendered in different fonts or font sizes. This contrasts eg. with `.multi_cell()`, where a change in font family or size can only become effective on a new line.
+
+
+#### write(h: float = None, txt: str = "", link: str = "", print_sh: bool = False) -> bool
+
+* __h__ (float): line height. If None, use the current font size.
+* __txt__ (str): text content
+* __link__ (str): optional link to add on the text, internal
+	(identifier returned by `add_link`) or external URL.
+* __print_sh__ (bool): Treat a soft-hyphen (\\u00ad) as a normal printable
+	character, instead of a line breaking opportunity.
+
+
+## .write_html()
+This method can be accessed by using the class `HTMLMixin` from "[html.py](HTML.html)". It is very similar to `.write()`, but accepts basic HTML formatted text as input. See [html.py](HTML.html) for more details and the supported HTML tags.
+
+Note that when using data from actual web pages, the result may not look exactly as expected, because `.write_html()` prints all whitespace unchanged as it finds them, while webbrowsers rather collapse each run of consequitive whitespace into a single space character.
+
+#### write_html(text, image_map=None, li_tag_indent=5, table_line_separators=False, ul_bullet_char=BULLET_WIN1252, heading_sizes=None)
```

Comment:
Just link to https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.write_html